### ISSUE TITLE
Remove overscroll behaviour

### ISSUE THREAD

**Issue by dubisdev** [kunkunsh/kunkun#75]
**Describe the bug**
See screenshot

**To Reproduce**
1. Open the app
2. Try to scroll over the app limits

**Expected behavior**
Do nothing

**Screenshots**

https://github.com/user-attachments/assets/9b73ac0e-fb70-42c3-ace1-a23418562314

**Desktop (please complete the following information):**
 - OS: Win 11
 - Browser: NA
 - Version: NA

**Additional context**

I think this is related to how each native webview executes css (maybe this only applies to Windows?)

I managed to get ride of this in other tauri apps by using

```css
* {
  overscroll-behavior: none;
}
```

**Comment by HuakunShen** [comment]
> I think this is related to how each native webview executes css (maybe this only applies to Windows?)

Right, I mainly develop this app on mac and have never seen this issue.
I will try to fix this with your solution on windows.

**Comment by HuakunShen** [comment]
I tried to overscroll on windows but failed. How did you do this? Are you using a trackpad? I only have mouse.

**Comment by HuakunShen** [comment]
Which version are you using? You can find it in settings -> about.
Can you try this latest version https://github.com/kunkunsh/kunkun/releases/tag/Kunkun-v0.1.9-beta.4
Although I don't think there is a difference.

**Comment by dubisdev** [comment]
> I tried to overscroll on windows but failed. How did you do this? Are you using a trackpad? I only have mouse.

Yes, I'm using the trackpad from my laptop.

I'm using 0.1.9-beta.2 (i didnt found a more recent executable)

**Comment by HuakunShen** [comment]
0.1.9-beta.4 was removed just now due to a serious bug.
I don't have a windows laptop, maybe you can try to run it on your leptop?

**Comment by dubisdev** [comment]
Hi @HuakunShen 

Just tested this in the new beta version (0.1.9-beta.6). Problem persists

**Comment by HuakunShen** [comment]
I didn’t try to fix this issue currently as I don’t have a windows with trackpad. 
I will install windows on a linux laptop then fix this issue.
Currently blocked with a more serious issue #79 which fails the entire extension loading system on windows.
